specification/cosmosdbmongocluster/resource-manager/Microsoft.DocumentDB/preview/2023-03-01-preview/mongoCluster.json

```diff
@@ -0,0 +1,1024 @@
+{
```

Comment:
There is already mongoCluster/json swagger under [https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-15-preview), why is this being added in a new folder structure?

Reply:
We are making a breaking change to remove the mongoCluster.json from the existing structure - I'm waiting on the last preview release for cosmos-db to be merge (2023-09-15-preview) and will add a change to remove it from their.

We are making this split so we can have separate API releases for Mongo clusters.